src/environment.js

```diff
@@ -3,15 +3,15 @@
 var path = require('path');
 var lib = require('./lib');
 var Obj = require('./object');
-var lexer = require('./lexer');
 var compiler = require('./compiler');
 var builtin_filters = require('./filters');
 var builtin_loaders = require('./loaders');
 var runtime = require('./runtime');
 var globals = require('./globals');
 var Frame = runtime.Frame;
+var Template, Environment;
 
-var Environment = Obj.extend({
+Environment = Obj.extend({
```

Comment:
I'm sorry, but I don't like these changes. Does it really complain about a `var` here? Can you make it so that it doesn't?